server/chat.ts

```diff
@@ -1940,10 +1940,7 @@ export const Chat = new class {
 			}
 			buf += '</span>';
 		}
-		let bst = 0;
-		for (const baseStat of Object.values(species.baseStats)) {
-			bst += baseStat;
-		}
+		let bst = species.bst;
```

Comment:
You can remove this entirely, this is a kind of silly alias. I'm also surprised that `prefer-const` isn't yelling at you.

Reply:
It isn't yelling at me since it gets modified later into the code

Reply:
https://github.com/smogon/pokemon-showdown/pull/7138/files#diff-14924f4ee018df5f23596233f35205d6R1949

Reply:
That should probably also be implemented in dex-data instead?

Reply:
There's no way to check what the dex's gen is in dex-data

Reply:
I suppose it should be implemented in dex, then?

Reply:
I don't see why we would do that

Reply:
I don't see why you would rather have an incorrect `species.bst` in Gen 1. That seems obviously worse to me.